api/path_node_userinput.go

```diff
@@ -39,6 +43,17 @@ func UpdateNodeUserInput(userInput []policy.UserInput,
 		return errorhandler(nil, NewNotFoundError("Exchange registration not recorded. Complete account and node registration with an exchange and then record node registration using this API's /node path.", "node")), nil, nil
 	}
 
+	// verify userinput: 1) service exist, 2) variables definied in the service, otherwise return true but give warning message 3) values have correct type
+	validated := false
+	for _, u := range userInput {
+		validated, err = ValidateUserInput(u, getService)
+		if !validated {
+			return errorhandler(nil, NewSystemError(fmt.Sprintf("Failed to validate node userInput, error: %v", err))), nil, nil
+		} else if err != nil { // validate == true, but give back warning error message
+			glog.Warningf(logString(fmt.Sprintf("Warning message: Userinput validated: %v. error: %v \n", validated, err)))
```

Comment:
Change to glog.Warningf(apiLogString(fmt.Sprintf("UPDATE node/userinput %v ", err)))